src/pymor/tools/floatcmp.py

```diff
@@ -31,7 +31,7 @@ def float_cmp(x, y, rtol=1e-14, atol=1e-14):
     atol
         The absolute tolerance.
     """
-    return np.abs(x - y) <= atol + np.abs(y) * rtol
+    return np.abs(x - y) <= atol + np.abs(y * rtol)
```

Comment:
I wonder if this fixes the problems we have in larger hypothesis configs https://github.com/pymor/pymor/issues/1741